src/downloadlistwidgetcompact.cpp

```diff
@@ -361,6 +366,7 @@ bool DownloadListWidgetCompactDelegate::editorEvent(QEvent *event, QAbstractItem
             }else {
               menu.addAction(tr("Visit on Nexus"), this, SLOT(issueVisitOnNexus()));
             }
+            menu.addAction(tr("Open File"), this, SLOT(issueOpenFile()));

```

Comment:
add checks to make sure the download is in the READY state. We don't want to give the user ability to open a still downloading file

Reply:
This `menu.addAction` is already inside an `if (state >= DownloadManager::STATE_READY)` clause, so I don't think we need to check for this again?